web_tool_script_3.R

```diff
@@ -163,6 +163,22 @@ js_translations <- jsonlite::fromJSON(
   txt = path(template_path, "data/translation/js_labels.json")
 )
 
+sector_order <- function(path) {
+  # TODO: Remove following create_interactive_report#313
+  if (!fs::file_exists(path)) {
+    rlang::warn(glue::glue(
+      "This file doesn't exit: {path}.
+      Have you already merged create_interactive_report#313?"
+    ))
+    return(NULL)
+  }
+
+  readr::read_csv(
+    path(template_path, "data", "sector_order", "sector_order.csv"),
+    col_types = cols()
+  )
+}
```

Comment:
This will now work even if the data is unavailable. This is an intermediate refactoring step to break the dependency on create_interactive_report#313.